reverse_engineering/api.js

```diff
@@ -316,8 +330,12 @@ const getBucketInfo = ({ _, metadata, datasetName }) => {
 	};
 };
 
-const getLabels = (_, labels) => {
-	return _.keys(labels).map(labelKey => ({ labelKey, labelValue: labels[labelKey] }));
+const getLabels = labels => {
+	if (!labels) {
+		return [];
+	}
```

Comment:
```suggestion
const getLabels = (labels = []) => {
```

Reply:
It won't work because `labels` can be `null`

Reply:
Got it.